tests/integration/test_k8s_with_cluster.py

```diff
@@ -46,28 +45,25 @@
 
 @mark.usefixtures("setup_cluster")
 def test_workflow_and_service_management_end_to_end_from_cli(
-    random_test_namespace: str, docker_image: str, ingress_load_balancer_url: str
+ docker_image: str, ingress_load_balancer_url: str
 ):
     try:
-        sleep(5)
-
         process_zero = run(
-            ["bodywork", "setup-namespace", random_test_namespace],
+            ["bodywork", "setup-namespace", "bodywork-test-project"],
```

Comment:
This means we can't both be running tests at the same time. Is this interim or something we need to live with for the time being?

Reply:
I'm referring to making the namespace deterministic.

Reply:
I see what you're doing now - very clever!

Pre-setup the namespace the project is deployed to, with the secrets loaded into it, until we sort secrets out.

NICE ONE!

Reply:
Yes because the namespace is derived from the config and we are using the same one (from Github). This was my point the other day about having a separate cluster for testing because we can now impact on live stuff we have running on our current cluster. 

When we have the secrets change done then we can just add a different namespace in the config locally and then we can avoid collisions with each other. This is fine for now as it's just us two.